### ISSUE TITLE
Test failing for "test_say"

### ISSUE THREAD

**Issue by anaisabel7** [sukeesh/Jarvis#89]
Right after cloning and installing dependencies, running tests with the command
`python -m unittest discover`
reports a failing test with this information:
`AssertionError: Lists differ: ['say', "i'm", 'a', 'robot'] != ['say', "i'm", 'a', 'robot?']`
(notice the ? sign)

I believe I understand why this happens and I am more than happy to solve it.
However, I am not 100% of what is the desired behaviour.

A different would be observed in behaviour if we had a single word sentence.

`can you say onions?`
would currently produce a slightly different pronunciation to
`say onions`
since the ? sign is understood as part of the sentence to be pronounced.
Notice that `can you say onions` and `say onions` produce the same output.

Basically, do I correct the test to expect the output from `can you say i'm a robot?` to be that of `say i'm a robot?` (instead of that of `say i'm a robot`) or do I change the code to ignore the last ? sign if the sentence of the say command starts with can or could?

Apologies for the long explanation.
